api/core/core.go

```diff
@@ -112,6 +113,129 @@ type (
 
 		// TODO time to live?
 	}
+
+	Status int32
+)
+
+// OpenCensus uses google.golang.org/grpc/ Following is copied from there.
```

Comment:
Why not use the codes from https://godoc.org/google.golang.org/grpc/codes then?

Reply:
I'll replace with grpc/codes.